core/trino-main/src/main/java/io/trino/sql/planner/SortExpressionExtractor.java

```diff
@@ -24,6 +24,7 @@
 import io.trino.sql.tree.Node;
```

Comment:
Could you add a change description instead of `Change-Id: Ifb4434a1babe9205461492a3572f6ec4aa6368cd`